docs/modules/ROOT/pages/how-tos/vsphere/pre-install-checklist.adoc

```diff
@@ -0,0 +1,175 @@
+For the A records of `etcd`, make sure that the SRV records are not created (or delete them, if they were), so the hostnames can be enumerated correctly.
+====
+
+=== Certificates
+
+Certificates for the following hostnames need to be created:
+
+* `api.ocp4-vs.appuio.ch`
+* `console.ocp4-vs.appuio.ch`
+* `*.apps.ocp4-vs.appuio.ch` (wildcard certificate)
+
+[WARNING]
+====
+For the certificates to be valid in modern web browser, make sure the _Subject Alternative Name (SAN)_ contains the hostname of the certificate.
+
+[[additional-ca]]
+If the certificates are issued from your own private PKI, the RootCA certificate of your PKI needs to be added to the `install-config.yaml` as described in the https://docs.openshift.com/container-platform/4.4/networking/configuring-a-custom-pki.html#installation-configure-proxy_configuring-a-custom-pki[documentation]:
+[source,bash]
+----
+...
+additionalTrustBundle: | 
+    -----BEGIN CERTIFICATE-----
+    <MY_TRUSTED_CA_CERT_1>
+    -----END CERTIFICATE-----
+    -----BEGIN CERTIFICATE-----
+    <MY_TRUSTED_CA_CERT_2>
+    -----END CERTIFICATE-----
+...
+----
+Multiple CA certificates can be added.
+====
+
+=== Load balancers
+
+For accessing API and applications, the following load balancers are required:
+
+* Console load balancer
+  ** Targets: all 3 masters as well as the bootstrap node at the beginning of the installation
+  ** Ports: 6443/tcp, 22623/tcp
+  ** TLS passthrough (no TLS termination on the load balancer
+  ** WebSockets allowed
+* Apps load balancer
+  ** Targets: all 3 Infra Nodes
+  ** Ports: 80/tcp, 443/tcp
+  ** TLS passthrough (optimally no TLS termination on the load balancer)
+
+=== Communication
+
+The following communication has to be guaranteed:
+
+|===
+|Source |Destination |Ports |Comment
+
+|OpenShift |Console load balancer |6443/tcp, 22623/tcp |All nodes
+
+|OpenShift |App load balancer |80/tcp, 443/tcp |All nodes
+
+|OpenShift |Outgoing proxy |depending on proxy |All nodes
+
+|OpenShift masters |Configured IdP |depending on IdP |Authentication is handled by components running on masters
+|===
+
+Possible further surrounding systems:
+
+* Access to Git repositories from all worker nodes
+* Access to databases and other applications from all worker nodes
+
+=== Outgoing proxy
+
+The following URLs need to be whitelisted on the outgoing proxy, as described in the https://docs.openshift.com/container-platform/4.4/installing/install_config/configuring-firewall.html[documentation].
+
+If the connection to the proxy is done via HTTPS, the CA certificate, which was used to sign the certificate of the proxy server, needs to be added to the `install-config.yaml`, as described <<additional-ca,above>>.
+
+See https://docs.openshift.com/container-platform/4.4/installing/installing_vsphere/installing-vsphere.html#installation-configure-proxy_installing-vsphere[Configuring the cluster-wide proxy during installation] for details.
```

Comment:
Placing the part about the CA in its own admonition, allows you to give it a title. You can then reference that title in the link text.

Reply:
https://asciidoctor.org/docs/user-manual/ holds an example of what I mean. Search for _Feeding the Werewolves_.